File: scratch/test_models.py
import urllib.request
import json

def test():
    try:
        url = 'https://god-maog.onrender.com/openai/v1/models'
        req = urllib.request.Request(url)
        with urllib.request.urlopen(req) as response:
            status = response.getcode()
            print('Status:', status)
            data = json.loads(response.read().decode())
            print('Data:', json.dumps(data, indent=2))
    except Exception as e:
        print('Error:', e)

if __name__ == '__main__':
    test()

File: scratch/test_models.js
async function test() {
  try {
    const res = await fetch('https://god-maog.onrender.com/openai/v1/models');
    console.log('Status:', res.status);
    const data = await res.json();
    console.log('Data:', JSON.stringify(data, null, 2));
  } catch (e) {
    console.error('Error:', e);
  }
}
test();
